pkg/metricsproxy/histogramVec_test.go

```diff
@@ -0,0 +1,41 @@
+package metricsproxy
+
+import (
+	"math/rand"
+
+	. "github.com/pingcap/check"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func (t *testMetricsProxySuite) TestHistogramVecProxy(c *C) {
+	for _, oneCase := range testCases {
+		histogram := NewHistogramVec(prometheus.HistogramOpts{
+			Namespace:   "dm",
+			Subsystem:   "metricsProxy",
+			Name:        "Test_Histogram",
+			Help:        "dm histogram metrics proxy test",
+			ConstLabels: nil,
+		}, oneCase.LabelsNames)
+		for _, aArgs := range oneCase.AddArgs {
+			if rand.Intn(199)%2 == 0 {
```

Comment:
Why `rand` is used here?

Reply:
Just to improve test coverage, so I used a rand here, or is there any other recommended way

Reply:
If you want to improve test coverage, `rand.Seed` should be called before use `rand.Intn`. Or the value of `rand.Intn` won't be updated.

Reply:
or maybe i need to test two case in differemt method?

Reply:
:joy: ok, got it

Reply:
I think that's also good.

Reply:
i fixed it